catalog/resource_metastore.go

```diff
@@ -0,0 +1,125 @@
+package catalog
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type MetastoresAPI struct {
+	client  *common.DatabricksClient
+	context context.Context
+}
+
+func NewMetastoresAPI(ctx context.Context, m interface{}) MetastoresAPI {
+	return MetastoresAPI{m.(*common.DatabricksClient), ctx}
+}
+
+type MetastoreInfo struct {
+	Name         string  `json:"name"`
```

Comment:
Should it be force new? Together with storage root?

Reply:
metastore is identified by UUID, so changing name is just fine. storage root, on the other hand, cannot be updated.

Reply:
yeah, I saw ForceNew later in the code, after I left the comment. Why not annotation? ;-)

Reply:
Oh, forgot about it! Will change for annotation.